packages/pigeon/lib/generator.dart

```diff
@@ -1,13 +1,121 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+// found in the LICENSE [indent].
```

Comment:
Oops :)

Reply:
totally meant to do that

Reply:
It's important that we verify from time to time that the license-check script is actually still working!